src/views/conference/2018/index/index.jsx

```diff
@@ -21,7 +21,6 @@ var ConferenceSplash = React.createClass({
                     <h1 className='title-banner-h1 mod-2018'>
                         <center>
                             <FormattedMessage id='conference-2018.title' />
-                            <br />
```

Comment:
My bad - my version still had two `<br />` I think we still need at least one to force the subtitle onto a new line.

Reply:
Okay, I'll add it back. Let me also resizze the image

Reply:
I think we have to adjust the line height of the `title-banner-h1.mod-2018`. We definitely need the `<br />` to separate the lines, and there's still a bit too much space between them. I think you can just take the line height from the mobile screen media query:
```
.title-banner-h1.mod-2018 {
        line-height: 1.25em;
    }
```
and just move that up so that it's the default for all screens/windows.